utils/rpms/daos.spec

```diff
@@ -554,6 +562,9 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
+* Thu May 11 2023 Lei Huang <lei.huang@intel.com> 2.3.107-3
```

Comment:
Since you have to push an update, you could update the date here to get it more accurate to when this will land.